docs/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,13 +1,8 @@
 ### What was the problem?
+This PR resolves #INSERT_ISSUE_NUMBER
 
 ### How did I solve it?
```

Comment:
And if the other questions do not make reference to `I` then maybe this could be more consistent this way `How was it solved?`